pkg/debug/spec.sh

```diff
@@ -541,7 +542,7 @@ for CAN in /sys/class/net/*; do
     cat <<__EOT__
     ${COMMA}
     {
-     "ztype": ${ZTYPE},
+     "ztype": "${ZTYPE}",
```

Comment:
Does line 594 (which is under verbose) need the same fix?

Reply:
No, because in this case `pci_to_ztype()` always return a numeric value...

Reply:
OK